src/plugins/data/public/ui/typeahead/_suggestion.scss

```diff
@@ -138,16 +138,21 @@ $kbnTypeaheadTypes: (
 }
 
 .kbnSuggestionItem__text {
-  flex-grow: 0; /* 2 */
-  flex-basis: auto; /* 2 */
   font-family: $euiCodeFontFamily;
-  width: 250px;
   overflow: hidden;
   text-overflow: ellipsis;
   padding: $euiSizeXS $euiSizeS;
   color: $euiTextColor;
 }
 
+.kbnSuggestionItem--withDescription {
```

Comment:
This is an additional improvement not related to the regression itself. 
This allow to stretch `text` to full width when description is not visible